woodwork/tests/accessor/test_statistics.py

```diff
@@ -271,15 +273,12 @@ def test_describe_accessor_method(describe_df):
         assert isinstance(stats_df, pd.DataFrame)
         assert set(stats_df.columns) == {'category_col'}
         assert stats_df.index.tolist() == expected_index
-        expected_vals.equals(stats_df['category_col'].dropna())
+        assert expected_vals.equals(stats_df['category_col'].dropna())
```

Comment:
Note - these lines were missing the `assert` so were never being evaluated in our tests before.